robolectric/src/test/java/org/robolectric/shadows/ShadowOpenGLMatrixTest.java

```diff
@@ -257,4 +285,282 @@ public void multiplyMVWithOffset() throws Exception {
+            5, 6, 7, 8,
+            9, 10, 11, 12,
+            13, 14, 15, 16
+    };
+    float[] expected = new float[]{
+            2, 4, 6, 8,
+            20, 24, 28, 32,
+            54, 60, 66, 72,
+            13, 14, 15, 16
+    };
+    Matrix.scaleM(matrix, 0, 2, 4, 6);
+    Assert.assertArrayEquals(expected, matrix, 0.001f);
+  }
+
+  @Test
+  public void testTranslateM() {
+    float[] matrix = new float[]{
+            1, 2, 3, 4,
+            5, 6, 7, 8,
+            9, 10, 11, 12,
+            13, 14, 15, 16
+    };
+    float[] expected = new float[]{
+            1, 2, 3, 4,
+            5, 6, 7, 8,
+            9, 10, 11, 12,
+            89, 102, 115, 128
+    };
+    float[] output = new float[16];
+    Matrix.translateM(output, 0, matrix, 0, 2, 4, 6);
+    Assert.assertArrayEquals(expected, output, 0.001f);
+  }
+
+  @Test
+  public void testTranslateMInPlace() {
+    float[] matrix = new float[]{
+            1, 2, 3, 4,
+            5, 6, 7, 8,
+            9, 10, 11, 12,
+            13, 14, 15, 16
+    };
+    float[] expected = new float[]{
+            1, 2, 3, 4,
+            5, 6, 7, 8,
+            9, 10, 11, 12,
+            89, 102, 115, 128
+    };
+    Matrix.translateM(matrix, 0, 2, 4, 6);
+    Assert.assertArrayEquals(expected, matrix, 0.001f);
+  }
+
+  @Test
+  public void testRotateM() {
+    float[] matrix = new float[]{
+            1, 2, 3, 4,
+            5, 6, 7, 8,
+            9, 10, 11, 12,
+            13, 14, 15, 16
+    };
+    float[] expected = new float[]{
+            0.95625275f, 1.9625025f, 2.968752f, 3.9750016f,
+            5.0910234f, 6.07802f, 7.0650167f, 8.052013f,
+            8.953606f, 9.960234f, 10.966862f, 11.973489f,
+            13, 14, 15, 16
+    };
+    float[] output = new float[16];
+    Matrix.rotateM(output, 0, matrix, 0, 2, 4, 6, 8);
+    Assert.assertArrayEquals(expected, output, 0.001f);
+  }
+
+  @Test
+  public void testRotateMInPlace() {
+    float[] matrix = new float[]{
+            1, 2, 3, 4,
+            5, 6, 7, 8,
+            9, 10, 11, 12,
+            13, 14, 15, 16
+    };
+    float[] expected = new float[]{
+            0.95625275f, 1.9625025f, 2.968752f, 3.9750016f,
+            5.0910234f, 6.07802f, 7.0650167f, 8.052013f,
+            8.953606f, 9.960234f, 10.966862f, 11.973489f,
+            13, 14, 15, 16
+    };
+    Matrix.rotateM(matrix, 0, 2, 4, 6, 8);
+    Assert.assertArrayEquals(expected, matrix, 0.001f);
+  }
+
+  @Test
+  public void testSetRotateM() {
+    float[] matrix = new float[]{
+            1, 2, 3, 4,
+            5, 6, 7, 8,
+            9, 10, 11, 12,
+            13, 14, 15, 16
+    };
+    float[] expected = new float[]{
+            0.9998687f, 0.01299483f, -0.00968048f, 0,
+            -0.012931813f, 0.999895f, 0.006544677f, 0,
+            0.009764502f, -0.006418644f, 0.99993175f, 0,
+            0, 0, 0, 1
+    };
+    Matrix.setRotateM(matrix, 0, 1, 2, 3, 4);
+    Assert.assertArrayEquals(expected, matrix, 0.001f);
```

Comment:
We prefer to use assertj where possible, can this be changed to

assertThat(matrix).containsSequence(expected, withPrecision(0.001f));